app/helpers/hyrax/collections_helper.rb

```diff
@@ -43,6 +43,29 @@ def collection_type_label(collection_type_gid)
       collection_type_gid
     end
 
+    def banner_file(collection_id)
+      # Find Banner filename
+      ci = CollectionBrandingInfo.where(collection_id: collection_id.to_s).where(role: "banner")
+      # @banner_file = File.split(ci[0].local_path).last unless ci.empty?
+      # @banner_file_location = ci[0].local_path unless ci.empty?
+      @banner_file_for_display = "/" + ci[0].local_path.split("/")[-4..-1].join("/") unless ci.empty?
```

Comment:
Why are we setting a instance variable here?

Reply:
I don't know, I didn't work on this. I will ask Lynette about it.